docs/streams/upgrade-guide.html

```diff
@@ -154,6 +154,14 @@ <h3><a id="streams_api_changes_270" href="#streams_api_changes_270">Streams API
         turn on all higher levels, ie INFO and DEBUG. See <a href="https://cwiki.apache.org/confluence/x/gBkRCQ">KIP-613</a> for more information.
     </p>
 
+    <p>
+        We added <code>prefixScan</code> API to <code>ReadOnlyKeyValueStore</code>. This would allow scanning the state store to fetch all
+        keys starting with the prefix supplied as an argument. Since the type of the key can be different from that of the key, this API also
+        takes a serializer as an argument to convert the prefix into the format in which the keys have been stored in the state store. See
+        <a href="https://cwiki.apache.org/confluence/display/KAFKA/KIP-614%3A+Add+Prefix+Scan+support+for+State+Stores">KIP-614</a> for
+        more details.
```

Comment:
You need to move this paragraph to line 124 since it will be released with 2.8 and NOT with 2.7.

I also have some suggestions:

```suggestion
        We added method <code>prefixScan()</code> to interface <code>ReadOnlyKeyValueStore</code>.
        Method <code>prefixScan()</code> allows fetching all values whose keys start with a given prefix. 
        See <a href="https://cwiki.apache.org/confluence/display/KAFKA/KIP-614%3A+Add+Prefix+Scan+support+for+State+Stores">KIP-614</a> for more details.
```